package.json

```diff
@@ -29,5 +30,8 @@
   },
   "jest": {
     "testEnvironment": "jsdom"
+  },
+  "dependencies": {
+    "clsx": "^1.1.1"
```

Comment:
Super tiny dependency, but I understand if you don't want the package to have any. We can create our own helper instead, just let me know.

Reply:
I’ve gone through the dependency and it looks very neat and tiny, so I’m all fine with keeping it. :)